File: app/src/main/java/top/gushenge/testdemo/TaobaoNews/NewsActivity.java
package top.gushenge.testdemo.TaobaoNews;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import butterknife.BindView;
import butterknife.ButterKnife;
import top.gushenge.testdemo.R;

public class NewsActivity extends AppCompatActivity {

    @BindView(R.id.news_list)
    AutoScrollView news;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_news);
        ButterKnife.bind(this);
        NewsAdapter adapter = new NewsAdapter();
        news.setAdapter(adapter);
    }
}

File: app/src/main/kotlin/com/gushenge/testdemo/TaobaoNews/NewsActivity.kt
package com.gushenge.testdemo.TaobaoNews

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

import butterknife.BindView
import butterknife.ButterKnife
import com.gushenge.testdemo.BaseActivity
import com.gushenge.testdemo.R
import kotlinx.android.synthetic.main.activity_news.*

class NewsActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_news)
        val adapter = NewsAdapter(this)
        news_list.adapter = adapter
    }
}
